src/components/LoginPage.jsx

```diff
@@ -26,7 +26,7 @@ const LoginPage = props => {
                 localStorage.setItem('jwtToken', res.data.token);
                 setAuthorizationToken(res.data.token);
 
-                props.history.push('/');
+                props.history.push('/dashboard');
```

Comment:
Please, do this same thing in the Signup component.

Reply:
I think it will be better if a user is directed to the login view after registration Since we might need to confirm email in the future. It is a normal practice in apps these days